### PR TITLE
fix polkadot crash

### PR DESCRIPTION
**Pull request by Pantani**
closes https://github.com/trustwallet/blockatlas/issues/767

Polkadot is crashing for two reasons because doesn't have the second object inside the slice data and sometimes can't convert the value two floats. Can you check if is this right @hewigovens ?

### CODE REVIEW

File: platform/polkadot/client.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ type Client struct {
 
 func (c *Client) GetTransfersOfAddress(address string) ([]Transfer, error) {
 	var res SubscanResponse
-	err := c.Post(&res, "/scan/transfers", TransfersRequest{Address: address, Row: blockatlas.TxPerPage})
+	err := c.Post(&res, "scan/transfers", TransfersRequest{Address: address, Row: blockatlas.TxPerPage})
```
Comment (hewigovens) [inline platform/polkadot/client.go]: I think we should remove / in `GetBase`? what if there is a trailing / in `config.yml`

Comment (Pantani) [reply platform/polkadot/client.go]: the method applies the `/`, we don't need them

File: platform/polkadot/api.go [hunk 2]
```diff
@@ -125,14 +125,21 @@ func (p *Platform) NormalizeExtrinsic(srcTx *Extrinsic) *blockatlas.Tx {
...
 	}
 
+	value := "0"
+	if len(datas) > 0 {
+		vf, ok := datas[0].Value.(float64)
+		if ok {
+			value = fmt.Sprintf("%.0f", vf)
+		}
+	}
 	decimals := p.Coin().Decimals
```
Comment (vikmeup) [inline platform/polkadot/api.go]: add test?

File: platform/polkadot/api.go [hunk 3]
```diff
@@ -125,14 +125,21 @@ func (p *Platform) NormalizeExtrinsic(srcTx *Extrinsic) *blockatlas.Tx {
 		Sequence: srcTx.Nonce,
 	}
 
+	value := "0"
+	if len(datas) > 0 {
```
Comment (vikmeup) [inline platform/polkadot/api.go]: would it make sense to have similar function as: https://github.com/trustwallet/blockatlas/pull/777/files#diff-f77bff57c22cf320c2a91afaaef92572R22

Comment (Pantani) [reply platform/polkadot/api.go]: the logic's a little different

File: platform/polkadot/api.go (hunk 2 above)
Comment (Pantani) [reply platform/polkadot/api.go]: done!

**Pull request merged**